### DISCUSSION TITLE
Is there a way to prioritize search engines?

### DISCUSSION THREAD

**Discussion by linlinxza** [searxng/searxng#3870] [Q&A] [answered]
Currently, I use Brave and Mojeek enabled. Mojeek isn't that accurate. Brave is. But Mojeek results seem to come up first. Is there a way to make certain search engines have less of a priority and others have the higher priority?

**Comment by return42** [comment, marked as answer]
You can set `weight:` in the [engine settings](https://docs.searxng.org/admin/settings/settings_engine.html).

**Reply by linlinxza** [reply]
So... The higher the number, the higher the priority?

**Reply by return42** [reply]
From the linked doc: "Weighting of the results of this engine."  .. increase the prio.
